patterns/1-initial/governance-levels.md

```diff
@@ -2,75 +2,72 @@
 
 Transparent Governance Levels
 
-## Context
+## Patlet
+
+Several teams are using different InnerSource patterns and all calling it "InnerSource", so the term is not precise enough to usefully describe a set of working practices without confusion.
 
-Several teams are using InnerSource patterns and best practices. However the
-degree to which they welcome not only contributions but give equal collaboration
-rights to contributors differ. As a result there are unmatched expectations,
-confusion and frustration when teams collaborate across team boundaries.
+Estabilishing a more accurate common language that is understood across all teams allows anyone to communicate intent or set expectations efficiently without ambiguity.
```

Comment:
Some of this text (maybe all of it?) could provide additional value in the `Rationale` section of the pattern.

Reply:
This is some insightful comparisons to opensource so it would be good to incorporate somehow, will take a crack at that later this week. I do stand by the "... the term is not precise enough to ..." language though because I don't think it misses the point: InnerSource is simply not a precise term and therefore needs additional qualifiers when discussing at anything other than a high level.

Reply:
I wanted to capture your comment here for posterity, so incorporated it into this PR in 5095397